README.md

```diff
@@ -39,6 +39,36 @@ let button = UIButton(type: .Custom)
 button.setBorder(width: 2, color: .redColor())
 ```
 
+## Safe area insets
+
+Taylor provides a backward compatible version of `safeAreaInsets` property for UIView. The original API is available on iOS 11 and above and the replacement fills the gap on older OS version (iOS 7 to 10 inclusively). It is an opt-in feature that requires to be enabled in order to work.
+
+To enable the feature, place this piece of code the earliest possible in your AppDelegate's `application:didFinishLaunching:withOptions` method:
+```swift
+UIViewController.enableCompatibilitySafeAreaInsets()
+```
+
+Your can now access the `compatibilitySafeAreaInsets` property from any UIView. On iOS 11 and above, this property returns `safeAreaInsets`. To receive safe area update callbacks, your view must conform to the `CompatibilitySafeAreaInsetsUpdate` protocol.
```

Comment:
I would have named it `safeArea`, shorter and still meaningfull.

Reply:
Meaningful in terms of what it does yes but I feel that there is a bit of information missing on how does it compare to `safeAreaInsets`. It could be a bit confusing for the end user as of witch property/callback to use. Unless the couple of code snippets I saw, this code does not override `safeAreaInsetsDidChange()` to call `compatibilitySafeAreaInsetsDidChange()` (because it lives in an extension) so the host application still have to implements both callbacks. `safeArea` seems like a good name to abstract the two properties in a custom subview.

Reply:
I agree 👍